### DISCUSSION TITLE
求: Digital Combat Simulator

### DISCUSSION THREAD

**Discussion by fpemud** [Gamuxorg/bbs#212] [建议和闲聊]
游戏名称：Digital Combat Simulator
游戏名称(英文)：Digital Combat Simulator
下载地址 :（如果有的话）：https://www.digitalcombatsimulator.com/en/

大游戏, windows游戏, steam proton上面没有, 估计wine支持情况也不好, 愿共同尝试和开发

https://appdb.winehq.org/objectManager.php?sClass=application&iId=14085
https://appdb.winehq.org/objectManager.php?sClass=application&iId=15635

**Comment by wangling12** [comment]
DCS有steamhttps://store.steampowered.com/app/223750/
并且看起来有人wine成功过https://www.youtube.com/watch?v=zA097F_0irI
steam版本下
protontricks 223750 vcrun2015
protontricks 223750 corefonts
之后启动会显示窗口，但是窗口内黑屏，等proton更新吧

**Comment by shouhuanxiaoji** [comment]
还没搞明白这游戏，看起来叫DCS 电子战机模拟器
看官网有很多系列，steam上并不热门 即使免费。
至于wine性能，要看用的图形处理技术 linux上表现。
具体可以到米聊群 50323947 聊聊

**Comment by fpemud** [comment]
是的，就是DCS(Digital Combat Simulator)

**Comment by shouhuanxiaoji** [comment]
之前做过几个游戏的wine解决方案，可以尝试一下。
